### PR TITLE
add Your rating column to applications list for committe users

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: app/imports/components/applications/applicationsLayout.jsx [hunk 1]
```diff
@@ -51,12 +51,13 @@ export class ApplicationsLayout extends Component {
                   <td className='s-4'><T>applications.title</T></td>
                   { this.props.isAdmin ? <td className='s-2'><T>applications.status.label</T></td> : false }
                   <td className='s-1'><T>applications.rating.average.label</T></td>
+                  { this.props.isAdmin ?  false: <td className='s-2'><T>applications.rating.userRating.label</T></td> }
```
Comment (varffs) [inline app/imports/components/applications/applicationsLayout.jsx]: should this spacing be `? false :` ?

File: app/imports/containers/applicationsContainer.jsx [hunk 2]
```diff
@@ -29,19 +30,43 @@ const composer = (props, onData) => {
 
         let applications = Applications.find({}).fetch();
 
-        applications = _.map(applications, (application) => {
-          let applicationUser = Meteor.users.findOne(application.userId);
+        let isAdmin = Roles.userIsInRole(user._id, 'admin') || false;
```
Comment (varffs) [inline app/imports/containers/applicationsContainer.jsx]: does `userIsInRole` not return false if not true?

File: app/imports/components/applications/applicationsLayout.jsx [hunk 3]
```diff
@@ -51,12 +51,13 @@ export class ApplicationsLayout extends Component {
...
                   { this.props.isAdmin ? <td className='s-5'><T>applications.actions.label</T></td> : false }
                 </tr>
               </thead>
               <tbody>
                 {this.props.applications.map((application, key) => (
-                  <ApplicationsApplication _id={application._id} userId={application.userId} userEmail={application.userEmail} isAdmin={this.props.isAdmin} galleryName={application.galleryName} status={application.status} key={key} averageRating={application.averageRating} />
+                  <ApplicationsApplication _id={application._id} userId={application.userId} userEmail={application.userEmail} isAdmin={this.props.isAdmin} galleryName={application.galleryName} status={application.status} key={key} averageRating={application.averageRating} userRating={application.userRating} />
```
Comment (varffs) [inline app/imports/components/applications/applicationsLayout.jsx:60]: `application.userRating` is never defined for the admin user. Is this a problem?

**Review by varffs — COMMENTED** [review]
I think the Sort label needs to change to `Average Rating` not just `Rating` now. Otherwise it works fine just my comments to check

File: app/imports/components/applications/applicationsLayout.jsx (hunk 3 above)
Comment (mixcas) [reply app/imports/components/applications/applicationsLayout.jsx:60]: yeah its ok

File: app/imports/containers/applicationsContainer.jsx (hunk 2 above)
Comment (mixcas) [reply app/imports/containers/applicationsContainer.jsx]: true

Comment (mixcas) [reply app/imports/containers/applicationsContainer.jsx]: lol, i mean u are right. ill fix it

**Pull request merged**